treescript/tests/test_gecko_android_l10n.py

```diff
@@ -0,0 +1,186 @@
+import os
+import shutil
+
+import pytest
+
+import treescript.gecko.android_l10n as android_l10n
+
+from unittest.mock import AsyncMock
+
+
+async def noop_async(*args, **kwargs):
+    pass
+
+
+# build_commit_message {{{1
+@pytest.mark.parametrize("dontbuild, ignore_closed_tree", ((True, True), (False, False)))
+def test_build_commit_message(dontbuild, ignore_closed_tree):
+    """build_commit_message adds the correct approval strings"""
+    expected = "no bug - DESCRIPTION r=release a=android_l10n-import"
+    if dontbuild:
+        expected += android_l10n.DONTBUILD_MSG
+    if ignore_closed_tree:
+        expected += android_l10n.CLOSED_TREE_MSG
+    assert android_l10n.build_commit_message("DESCRIPTION", dontbuild=dontbuild, ignore_closed_tree=ignore_closed_tree).rstrip() == expected
+
+
+# get_android_l10n_files_toml {{{1
+def test_get_android_l10n_files_toml(mocker):
+    toml_contents = """
+basepath = "."
+
+locales = [
+  "ab",
+  "en-GB",
+]
+
+[env]
+
+[[paths]]
+  reference = "app/src/main/res/values/strings.xml"
+  l10n = "app/src/main/res/values-{android_locale}/strings.xml"
+"""
+    open_mock = mocker.mock_open(read_data=toml_contents)
+    mocker.patch("builtins.open", open_mock, create=True)
+    mocker.patch.object(os.path, "exists", return_value=True)
+
+    assert android_l10n.get_android_l10n_files_toml("path") == []
```

Comment:
Should there be a test where get_android_l10n_files_toml does return something?

Reply:
I've added some additional test cases. Thanks.